src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderUtils.ts

```diff
@@ -154,14 +154,14 @@ export const getOperators = (
 };
 
 export const buildQuery = (querySnippets: QueryRowData[]) => {
-    let fullQuery = `SELECT *\nFROM DIGITALTWINS T\n`;
+    let fullQuery = `SELECT TOP(1000)\nFROM DIGITALTWINS T\nWHERE `;
```

Comment:
please use the constant here that you use to check for 1000 rows in the response.